config/locales/en.yml

```diff
@@ -42,7 +43,8 @@ en:
 
   geocoder_options: "Geocoder Options"
 
-  gtt_tile_sources_info: "Select the tile sources that should be available in this project."
+  gtt_tile_sources_info: "Select the tile sources that should be available in this\
+    \ project."
```

Comment:
@dkastl Are these `en.yml` new lines (with escape back slashes) necessary ?

Reply:
(I forgot to add comment, before resolving the conversation...)
Okay, thanks for confirmation.
If it comes from Weblate, keeping as it is (with new lines) may be enough.